src/tx/applyTx/checkSpendCond.js

```diff
@@ -403,6 +413,15 @@ module.exports = async (state, tx, bridgeState, nodeConfig = {}) => {
       } else {
         toAddr = `0x${toAddr.toString('hex')}`;
       }
+
+      // todo: support transfer of ERC1948
+      if (!isNFT(originColor) && data.length === 0) {
```

Comment:
I'm a bit unsure about this line. To we really cover all problems? 🤔

Reply:
i guess not. don't see that we have transfer of ERC1948 covered